bigbluebutton-html5/imports/ui/components/app/component.jsx

```diff
@@ -79,6 +79,11 @@ class App extends Component {
     document.getElementsByTagName('html')[0].lang = locale;
     document.getElementsByTagName('html')[0].style.fontSize = this.props.fontSize;
 
+    const BROWSER_RESULTS = browser();
+    const body = document.getElementsByTagName('body')[0];
+    body.classList.add('browser-' + BROWSER_RESULTS.name);
+    body.classList.add('os-' + BROWSER_RESULTS.os.split(' ')[0].toLowerCase());
```

Comment:
`BROWSER_RESULTS.os.split(' ').shift().toLowerCase()

Reply:
what is the benifit of using `BROWSER_RESULTS.os.split(' ').shift().toLowerCase()` over `BROWSER_RESULTS.os.split(' ')[0].toLowerCase()`  in this particular case?